cylc/flow/network/graphql.py

```diff
@@ -44,7 +45,7 @@
 
 STRIP_ARG = 'strip_null'
 NULL_VALUE = None
-EMPTY_VALUES = ([], {})
+EMPTY_VALUES: Tuple[list, dict] = ([], {})
```

Comment:
Is it possible to be more specific here, e.g. `Tuple[List[int], Dict[str, str]]` or whatever it's meant to be?